kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -1119,6 +1121,13 @@ def explore_args(self) -> ArgumentParser:
             action='store_true',
             help='Store a node for every EVM basic block (implies --break-on-calls).',
         )
+        args.add_argument(
+            '--break-on-load-program',
+            dest='break_on_load_program',
+            default=None,
+            action='store_true',
+            help='Store a node every time the jumpDests are computed, allowing the custom step to apply.',
```

Comment:
Ah, sorry, could we be a bit more informative here, what is the custom step now? Why would we want that, actually?

Reply:
Done!